co2eq/food/ingredients.js

```diff
@@ -1,29 +1,18 @@
 import {
-  MEAL_TYPE_VEGAN,
-  MEAL_TYPE_VEGETARIAN,
-  MEAL_TYPE_PESCETARIAN,
-  MEAL_TYPE_MEAT_LOW,
-  MEAL_TYPE_MEAT_MEDIUM,
-  MEAL_TYPE_MEAT_HIGH,
-  MEAL_TYPE_MEAT_OR_FISH,
   PURCHASE_CATEGORY_FOOD,
   UNIT_KILOGRAMS,
-} from '../definitions';
+} from '../../definitions';
 import {
   getEntryByKey,
   getDescendants,
   getEntryByPath,
   getChecksumOfFootprints,
-} from './purchase';
+} from '../purchase';
 
-const MEALS_PER_DAY = 3;
-
-// ** modelName must not be changed. If changed then old activities will not be re-calculated **
-export const modelName = 'meal';
-export const modelVersion = `5_${getChecksumOfFootprints()}`; // This model relies on footprints.yaml
+export const modelName = 'model name';
+export const modelVersion = `1_${getChecksumOfFootprints()}`;
 export const explanation = {
-  // TODO(olc): Write a description for mealType as well.
-  text: 'The calculations take into consideration emissions across the whole lifecycle.',
+  text: 'The calculations take into consideration emissions of greenhouse gases across the whole lifecycle of the ingredients.',
   links: [
     { label: 'Nature (2017)', href: 'https://www.nature.com/articles/s41598-017-06466-8' },
```

Comment:
not sure this is applicable anymore

Reply:
@martincollignon just updated that yesterday if I'm right